Author: Paul Wise <pabs@debian.org>
Description: Use -rdynamic instead of -export_dynamic since gcc does not understand it
--- pdp.orig/Makefile
+++ pdp/Makefile
@@ -26,11 +26,11 @@
 
 linux: pdp_all
 	rm -f pdp.pd_linux
-	$(CC) -export_dynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS)
+	$(CC) -rdynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS)
 
 linux_mmx: pdp_all
 	rm -f pdp.pd_linux
-	$(CC) -export_dynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS)
+	$(CC) -rdynamic -shared -o pdp.pd_linux modules/*/*.o system/pdp.o system/*/*.o puredata/*.o $(PDP_LIBS)
 
 buildclean:
 	make -C include clean
--- pdp.orig/opengl/Makefile
+++ pdp/opengl/Makefile
@@ -20,7 +20,7 @@
 
 pdp_opengl.pd_linux: subdirs
 	rm -f pdp_opengl.pd_linux
-	$(CC) -export_dynamic -shared -o pdp_opengl.pd_linux modules/*.o system/*.o $(LDFLAGS) -g
+	$(CC) -rdynamic -shared -o pdp_opengl.pd_linux modules/*.o system/*.o $(LDFLAGS) -g
 
 pdp_opengl.pd_darwin: subdirs
 	rm -f pdp_opengl.pd_linux
--- pdp.orig/scaf/Makefile
+++ pdp/scaf/Makefile
@@ -26,7 +26,7 @@
 
 pdp_scaf.pd_linux: pdp_scaf_all
 	rm -f pdp_scaf.pd_linux
-	gcc -export_dynamic -shared -o pdp_scaf.pd_linux pdp/*.o $(PDP_CA_LIBS)
+	gcc -rdynamic -shared -o pdp_scaf.pd_linux pdp/*.o $(PDP_CA_LIBS)
 
 install: all
 	install -d $(prefix)/lib/scaf
--- pdp.orig/scaf/compiler/scafc
+++ pdp/scaf/compiler/scafc
@@ -38,7 +38,7 @@
 
 TMP_S=`tempfile -s .s`
 $SCAFDIR/scafc.pl -I$SCAFDIR $1 > $TMP_S \
-&& gcc -export_dynamic -shared -o $DEST $TMP_S \
+&& gcc -rdynamic -shared -o $DEST $TMP_S \
 && strip --strip-unneeded $DEST \
 && rm $TMP_S \
 || exit 1
